package.json

```diff
@@ -23,6 +23,7 @@
   },
```

Comment:
Yes, thanks. It is not installed globally, I have installed it in the `node_modules` folder to make some local tests. And now I forgot to include it in the devDependencies.